helper/user-lookup/user-lookup.go

```diff
@@ -0,0 +1,33 @@
+package userlookup
```

Comment:
This should have some build flags to make sure it is only compiled on `!windows`